isos/bootstrap.sh

```diff
@@ -131,4 +162,5 @@ echo Total tempfs size: ${size}
 echo "${tempfs_target_list[@]}" > $(rootfs_dir $PKGDIR)/.tempfs_list
 echo ${size} > $(rootfs_dir $PKGDIR)/.tempfs_size
 
-generate_iso $PKGDIR $BIN/$ISONAME /lib/systemd/systemd
+INIT=$(cat $REPODIR/repo-spec.json | jq -r '.init')
+generate_iso $PKGDIR $BIN/$ISONAME $INIT
```

Comment:
We should ensure this deletes the tempdir after clean exit if DEBUG isn't set.

Reply:
Everything runs in a docker container so it's all wiped any way.